patroni/ha.py

```diff
@@ -94,6 +94,11 @@ def check_mode(self, mode):
         else:
             return self.patroni.config.check_mode(mode)
 
+    def master_stop_timeout(self):
+        """ Master stop timeout """
+        ret = parse_int(self.patroni.config.dynamic_configuration.get('master_stop_timeout'))
+        return ret if ret and ret > 0 and self.is_synchronous_mode() else None
+
```

Comment:
Please take a look how `master_start_timeout` works, you need to "declare" in the [config](https://github.com/zalando/patroni/blob/master/patroni/config.py#L61) and later you can access it as `self.patroni.config['master_stop_timeout']`